config/quality/checkstyle/checkstyle-config.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC
     "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
-    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd"

```

Comment:
As these changes does not belongs to this PR so please revert back.

Reply:
I don't change anything over here. I just put this ">" in the next line by mistake. But I will definitely change it.

Reply:
As of now, I don't know kotlin but I will definitely look into it soon.